BlockchainTests/ValidBlocks/bcStateTests/TransactionToItselfNotEnoughFounds.json

```diff
@@ -125,6 +125,30 @@
                         "value" : "0xc351"
                     }
                 ],
+                "transactionSequence" : [
```

Comment:
I think you pulled my changes here

Reply:
Oops, yes. I'll just close this PR and create a new, clean one.